src/components/pages/coach/CoachDashboard/index.jsx

```diff
@@ -109,115 +622,160 @@ class CoachDashboard extends React.Component{
         return selected;
     }
 
+    removeCustomer = (uid) => {
+        console.log("Called remove Customer");
+        clientNoCache.mutate({
+            mutation: REMOVE_CUSTOMER,
+            variables: { 
+                "token": localStorage.getItem("fprint"),
```

Comment:
"fprint" doesn't seem to be the best naming for an JWT in local storage.
I would prefer to name it anonymousJWT or something along those lines,
cause it's the JWT of the anonymous user.

Reply:
I have to agree on that one. They are a remenant of darker times. 🌋